test/e2e/framework/framework.go

```diff
@@ -61,6 +61,20 @@ func beforeSuite() {
 	for _, restConfig := range framework.RestConfigs {
 		MCSClients = append(MCSClients, createLighthouseClient(restConfig))
 	}
+
+	framework.By("Fetching ClusterIDs")
+
+	for idx, kubeClient := range framework.KubeClients {
+		agentDeployment, err :=
+			kubeClient.AppsV1().Deployments(framework.TestContext.SubmarinerNamespace).Get("submariner-lighthouse-agent", metav1.GetOptions{})
+		if err == nil {
```

Comment:
I assume we should not expect any error.

```suggestion
		Expect(err).To(Succeed())
```